src/common/tracer.h

```diff
@@ -0,0 +1,66 @@
+#ifndef TRACER_H_
+#define TRACER_H_
+
+#define SIGNED_RIGHT_SHIFT_IS 1
+#define ARITHMETIC_RIGHT_SHIFT 1
+
+#include <iostream>
+#include <arpa/inet.h>
+#include <yaml-cpp/yaml.h>
+
+#include <jaegertracing/Tracer.h>
+
+#ifndef WITH_JAEGER
+  #define WITH_JAEGER
```

Comment:
why do you do that?
this file should not be used at all if `WITH_JAEGER` is not defined

Reply:
it is just for testing purposes, it will be removed